lib/proc/proc_tokenize.c

```diff
@@ -54,6 +54,35 @@ fin_tokens(grn_ctx *ctx, grn_obj *tokens)
   GRN_OBJ_FIN(ctx, tokens);
 }
 
+static void
+output_only_tokens(grn_ctx *ctx, grn_obj *tokens, grn_obj *lexicon)
+{
+  size_t i, n_tokens, n_elements;
+  n_tokens = GRN_BULK_VSIZE(tokens) / sizeof(tokenize_token);
+  n_elements = 1;
+
+  grn_ctx_output_array_open(ctx, "TOKENS", (int)n_tokens);
+  for (i = 0; i < n_tokens; i++) {
+    tokenize_token *token;
+    token = ((tokenize_token *)(GRN_BULK_HEAD(tokens))) + i;
```

Comment:
```suggestion
    tokenize_token *token = ((tokenize_token *)(GRN_BULK_HEAD(tokens))) + i;
```

How about declaring and assigning at the same time?